nmdc_server/models.py

```diff
@@ -273,18 +273,6 @@ def open_in_gold(self) -> Optional[str]:
             self.gold_study_identifiers,  # type: ignore
         )
 
-    @property
-    def award_dois(self) -> list[DOIInfo]:
-        return [d for d in self.dois if d.doi_type == DOIType.AWARD]  # type: ignore
-
-    @property
-    def publication_dois(self) -> list[DOIInfo]:
-        return [d for d in self.dois if d.doi_type == DOIType.PUBLICATION]  # type: ignore
-
-    @property
-    def dataset_dois(self) -> list[DOIInfo]:
-        return [d for d in self.dois if d.doi_type == DOIType.DATASET]  # type: ignore
-
```

Comment:
What's the reason for getting rid of these properties? I think they'd still work based on the changes you've made. Are you thinking its best to just leave it to consumers of the API to sort out all the DOIs?

Reply:
Yes, that was my thought